Update the ImageMagick install documentation (#3807)
owncloud/docs: modules/admin_manual/pages/installation/manual_installation/manual_imagick7.adoc

sudo apt remove imagemagick-6-common
----

== Install ImageMagick 7

=== Install the Lastest ImageMagick-7 Binary

> To install ImageMagick-7, a script is used. Alternatively, you can copy&paste all installation commands step by step from {imei-url}[IMEI - ImageMagick Easy Install]. See the README description for further parametrisation. IMEI uses {checkinstall-url}[Checkinstall] for ease of removing/uninstalling ImageMagick 7 and its components.

Reviewer: ```suggestion
To install ImageMagick-7, a script is used. Alternatively, you can copy&paste all installation commands step by step from {imei-url}[IMEI - ImageMagick Easy Install]. See the README description for more information on options and parameters. IMEI uses {checkinstall-url}[Checkinstall] for ease of removing/uninstalling ImageMagick 7 and its components.
```

+
[source,console]
----
php -r 'phpinfo();' | grep -i "ImageMagick supported formats"
----

== TIPS

=== Reinstall or upgrade ImageMagick-7 and the php wrapper.

> To reinstall or upgrade ImageMagick-7, follow the principle steps described above by disabling and removing the php wrapper first, rerun the imei.sh installation script with the options of choice. The script checks if components need an upgrade and installs them in case. Then reinstall the php wrapper, enable it and restart your webserver or php-fpm.

Reviewer: ```suggestion
To reinstall or upgrade ImageMagick-7, follow the principle steps described above by disabling and removing the php wrapper first, rerun the imei.sh installation script with the options of choice. The script checks if components need an upgrade and, if that's the case, installs them. Then reinstall the php wrapper, enable it and restart your web server or php-fpm.
```

=== Uninstall ImageMagic-7 or components

If you want to uninstall ImageMagick-7 only, run:

[source,console]
----
sudo apt remove imei-imagemagick
----

If you want to completely remove ImageMagic-7 and all of its installed components, run:

[source,console]
----
sudo apt remove imei-imagemagick,imei-libaom,imei-libheif,imei-libjxl
----

> === Change Configuration Settings of ImageMagic-7

Reviewer: ```suggestion
=== Change Configuration Settings of ImageMagick-7
```

> You can change configuration settings of ImageMagick-7 at any time accoring your needs. In case you do so, restart your webserver of the php-fpm service post changing the settings so they can take effect for web services.

Reviewer: ```suggestion
You can change configuration settings of ImageMagick-7 at any time according to your needs. In case you do so, restart your web server of the php-fpm service post changing the settings so they can take effect for web services.
```
